repo/packages/A/arangodb3/5/config.json

```diff
@@ -0,0 +1,235 @@
+          "description": "Minimal mesos resources for an agent",
+          "type": "string",
+          "default": "mem(*):2048;cpus(*):0.25;disk(*):2048"
+        },
+
+        "minimal-resources-coordinator": {
+          "description": "Minimal mesos resources for a coordinator",
+          "type": "string",
+          "default": "mem(*):4096;cpus(*):1;disk(*):1024"
+        },
+
+        "minimal-resources-dbserver": {
+          "description": "Minimal mesos resources for a DBserver",
+          "type": "string",
+          "default": "mem(*):4096;cpus(*):1;disk(*):4096"
+        },
+
+        "minimal-resources-secondary": {
+          "description": "Minimal mesos resources for a secondary DBserver",
+          "type": "string",
+          "default": "mem(*):4096;cpus(*):1;disk(*):4096"
+        },
+
+        "nr-agents": {
+          "description": "Number of Agents",
+          "type": "integer",
+          "default": 3
+        },
+
+        "nr-dbservers": {
+          "description": "Initial number of DBservers",
+          "type": "integer",
+          "default": 2
+        },
+
+        "nr-coordinators": {
+          "description": "Initial number of coordinators",
+          "type": "integer",
+          "default": 2
+        },
+
+        "failover-timeout": {
+          "description": "Timeout after which an automatic failover is done.",
+          "type": "number",
+          "default": 604800
+        },
+
+        "mesos-authenticate": {
+          "description": "If true use authentication with Mesos master.",
+          "type": "boolean",
+          "default": false
+        },
+
+        "secret": {
+          "description": "Secret for authentication with Mesos.",
+          "type": "string",
+          "default": ""
+        },
+
+        "secondaries-with-dbservers": {
+          "description": "Flag, if each secondary must run on a node with a DBServer",
+          "type": "boolean",
+          "default": false
+        },
+
+        "coordinators-with-dbservers": {
+          "description": "Flag, if each coordinator must run on a node with a DBServer",
+          "type": "boolean",
+          "default": false
+        },
+
+        "arangodb-docker-image": {
+          "description": "Docker image to run arangodb.",
+          "type": "string",
+          "default": "arangodb/arangodb-mesos:3.1"
+        },
+
+        "arangodb-privileged-image": {
+          "description": "Run arangodb image in privileged mode.",
+          "type": "boolean",
+          "default": false
+        },
+
+        "arangodb-force-pull-image": {
+          "description": "Forcefully pull arangodb image.",
+          "type": "boolean",
+          "default": true
+        },
+
+        "arangodb-ssl-keyfile": {
+          "description": "Enable SSL using this base64 encoded SSL keyfile (see ArangoDB docs for keyfile specification)",
+          "type": "string",
+          "default": ""
+        },
+        "arangodb-enterprise-key": {
```

Comment:
please add a note about `arangodb-ssl-keyfile` and `arangodb-enterprise-key` and how to use them in dcos/examples for arangoDB